GoogleForCreators/web-stories-wp
# FTUE: Add "Get Started" story for users to view in the editor

## Summary

This PR implements the new "Get Started" story for users to view in the editor.

## Relevant Technical Choices

Somewhat elegant yet hacky implementation in `Stories_Controller` to check for existence of a GET param to ensure the preloaded REST API request will respond with the demo data.

## To-do

* [x] Update with final content
* [x] Remove unneeded sizes from resources
* [x] Update image URLs on the fly
* [x] Move images to CDN

## User-facing changes

New "Get Started" story can be access upon plugin activation.

Screenshot:

<img width="1680" alt="Screenshot 2020-09-16 at 14 35 08" src="https://user-images.githubusercontent.com/841956/93337738-d474e680-f829-11ea-869d-96a88a550dad.png">


## Testing Instructions

**Note:** for quick access, head directly to `/wp-admin/post-new.php?web-stories-demo=1&post_type=web-story`

1. Deactivate plugin.
1. Activate plugin
1. Click on link to "Get Started" story to access FTUE story

Things to verify:

1. title field is pre-filled
1. the content is pre-filled with 9 pages
1. no `L10N_PLACEHOLDER` placeholders are visible
1. all images and videos load correctly
1. story can be previewed & saved as expected

------

Fixes #4137

See #2909
